stix_shifter_modules/gcp_chronicle/configuration/lang_en.json

```diff
@@ -0,0 +1,24 @@
+{
+    "connection": {
+        "host": {
+            "label": "Management IP address or Hostname",
+            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
+        },
+        "help": {
+            "label": "Need additional help?",
+            "description": "More details on the data source setting can be found in the specified link"
+        },
+         "selfSignedCert": {
+                "label": "Private Key - Mandatory Field",
```

Comment:
Private Key (Required)

Reply:
Updated the label in lang_en.json as suggested